### PR TITLE
docs: updated workbench cognito documentation

### PR DESCRIPTION
**Pull request by bhudnell**
Issue #, if available:

Description of changes:
updated workbench cognito documentation

Checklist:

* [ ] Have you successfully deployed to an AWS account with your changes?
* [ ] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: workbench-core/infrastructure/docs/workbenchCognito.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  }
+}
+
+// Set the cognito properites
+// If a user pool name or user pool client name are not provided, CDK will generate them for you
+const cognitoProps: WorkbenchCognitoProps = {
+  domainPrefix: 'myDomainPrefix',
+  websiteUrls: ['https://mysite.mydomain.com'],
+  // Include your created OidcProvider if including a SSO component
+  oidcIdentityProviders = [ myWorkbenchUserPoolOidcIdentityProvider ]
```
Comment (nguyen102) [inline workbench-core/infrastructure/docs/workbenchCognito.md:51]: `accessTokenValidity` is also an option

Comment (bhudnell) [inline workbench-core/infrastructure/docs/workbenchCognito.md:51]:
```suggestion
  oidcIdentityProviders = [ myWorkbenchUserPoolOidcIdentityProvider ]
  // If an access token validity of 15 minutes isnt sufficient, you mach change it here
  accessTokenValidity: Duration.minutes(60) // 1 hour
```

File: workbench-core/infrastructure/docs/workbenchCognito.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  // If an access token validity of 15 minutes isnt sufficient, you mach change it here
```
Comment (bhudnell) [inline workbench-core/infrastructure/docs/workbenchCognito.md:52]:
```suggestion
  // If an access token validity of 15 minutes is insufficient, you may change it here
```

File: workbench-core/infrastructure/docs/workbenchCognito.md (hunk 1 above)
Comment (bhudnell) [reply workbench-core/infrastructure/docs/workbenchCognito.md:51]: added

File: workbench-core/infrastructure/docs/workbenchCognito.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  issuerUrl: 'providerIssuerUrl',
+  attributeMapping: {
+    // These attributes are used by workbench-core-authentication package
+    email: ProviderAttribute.other('providerEmailAttributeValue'),
+    givenName: ProviderAttribute.other('providerGivenNameAttributeValue'),
+    familyName: ProviderAttribute.other('providerFamilyNameAttributeValue'),
+  }
+}
+
+// Set the cognito properites
```
Comment (htztomic) [inline workbench-core/infrastructure/docs/workbenchCognito.md:45]:
```suggestion
// Set the cognito properties
```

**Review by htztomic — APPROVED** [review]

**Pull request merged**